individual/utils.py

```diff
@@ -0,0 +1,9 @@
+import uuid
+
+
+def is_valid_uuid(s):
```

Comment:
Can we move to the core? What do you think @malinowskikam . It's not only for this module, but also another modules can benefit from that. 
Also I think it's a kind of validation (can be used in validation.py?). 
Moreover what does mean 's' in argument? I assume 'string'. But it could good to expand the naming.

Reply:
I'll move it to core